inc/admin/networkmanagers/namespace.php

```diff
@@ -91,11 +91,8 @@ function update_admin_status() {
 
 		if ( is_array( $restricted ) && ! empty( $restricted ) ) {
 			update_site_option( 'pressbooks_network_managers', $restricted );
-			// This meta option will only be used by the BI, but we want to keep that in sync.
-			update_site_option( 'pressbooks_network_managers_ids', implode( ',', $restricted ) );
```

Comment:
I opted for removing the update directly here and rely only on the cron to update the value